module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -58,6 +59,13 @@ abstract class MinkTestCase extends \PHPUnit\Framework\TestCase
      */
     protected $modifiedConfigs = [];
 
+    /**
+     * Modified yaml configurations
+     *
+     * @var array
+     */
+    protected $modifiedYamlConfigs = [];
```

Comment:
This YAML functionality is a nice addition; maybe we should commit it separately from the rest. I can merge it early to dev if you like, just to reduce files in this PR.

Reply:
Sure, please do!

Reply:
Done: commit c906c31584dcdb8b33eb3bfc804159ee8e89b30d.